share/pvss/pvss.go

```diff
@@ -0,0 +1,182 @@
+// Package pvss implements public verifiable secret sharing as introduced in
+// "A Simple Publicly Verifiable Secret Sharing Scheme and its Application to
+// Electronic Voting" by Berry Schoenmakers. In comparison to regular verifiable
+// secret sharing schemes, PVSS enables any third party to verify shares
+// distributed by a dealer using zero-knowledge proofs. PVSS runs in three steps:
+// (1) The dealer creates a list of encrypted public verifiable shares using
```

Comment:
This will not show up correctly on godoc. Please use:
```
//  1. The...
```
with at least two spaces, else it goes together with the previous line.